DevelopGitHub/GitHubExtension/GitHubExtension.Notes.WebApi/IdentityExtensions.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Security.Claims;
+using GitHubExtension.Notes.WebApi.ViewModels;
+using Microsoft.AspNet.Identity;
+
+namespace GitHubExtension.Notes.WebApi
+{
+    public static class IdentityExtensions
+    {
+        public static string GetUserId()
+        {
+            var userId = ClaimsPrincipal.Current.Identity.GetUserId();
+            return userId;
+        }
+
+        public static NoteModel AddUserIdToModel (this NoteModel noteModel)
+        {
+            var userId = ClaimsPrincipal.Current.Identity.GetUserId();
```

Comment:
Idea here is that I do not pass user ID from UI in a model, but use extension method to get user ID from identity and add it to a note model. Then I return ready model to controller for mapping to Entity.